tests/e2e-leg-3/non-cluster-admin-deployment/45-assert.yaml

```diff
@@ -0,0 +1,19 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
```

Comment:
Shouldn't you remove steps 45 and 50 in your other PR?

Reply:
It's not a big deal. We can deal with conflict easy enough.

Reply:
I believe there will be no conflicts when we merge from vnext to main, as I put the step for vrep last, so that the step numbers for vrpq and vscr are the same in 2 branches